logs/querysample/querysample.go

```diff
@@ -31,6 +32,21 @@ func TransformAutoExplainToQuerySample(logLine state.LogLine, explainText string
 func transformExplainJSONToQuerySample(logLine state.LogLine, explainText string, queryRuntimeMs float64) (state.PostgresQuerySample, error) {
 	var explainJSONOutput state.ExplainPlanContainer
 
+	// With Heroku + JSON format, the value of "Query Text" can contain \n (new line).
+	// JSON's value can't contain \n, so replacing it with a white space
+	if cUtil.IsHeroku() {
```

Comment:
We can technically expand this outside of Heroku too (I think we have seen this invalid JSON with the other cloud provider before too...), but for now, I'd like to minimize the change.

Reply:
Our of curiosity, what was the error you were seeing? This sounds close to an issue I had last week trying to get EXPLAIN plans to work with rsyslog and couldn't resolve ATM and wanted to revisit.

Reply:
It's: Invalid character '\n' in string literal

Reply:
Yeah - that was the same error I was getting with rsyslog messages and I couldn't get around it. 🤔